src/Umbraco.Tests/FrontEnd/UmbracoHelperTests.cs

```diff
@@ -78,5 +93,76 @@ public void Create_Encrypted_RouteString_From_Dictionary()
             
             Assert.AreEqual(expectedResult, result);
         }
+
+        [Test]
+        public void Truncate_By_Words_TagsAsContentOff()
+        {
+            var text = "Hello world, this is some text <a href='blah'>with a link</a>";
+
+            var helper = new UmbracoHelper();
+
+            var result = helper.TruncateByWords(text, 4, true, false).ToString();
+
+            Assert.AreEqual("Hello world, this is&hellip;", result);
```

Comment:
Is this test testing the correct behaviour? There is HTML in this string but we are truncating before even reaching the HTML. Or maybe there's test missing for testing the HTML stripping behaviour?